src/transformers/activations.py

```diff
@@ -41,6 +37,24 @@ def gelu_fast(x):
     return 0.5 * x * (1.0 + torch.tanh(x * 0.7978845608 * (1.0 + 0.044715 * x * x)))
 
 
+def silu(x):
+    """
+    See Gaussian Error Linear Units (Hendrycks et al., https://arxiv.org/abs/1606.08415) 
+    where the SiLU (Sigmoid Linear Unit) was originally introduced and coined,
+    and see Sigmoid-Weighted Linear Units for Neural Network Function Approximation 
+    in Reinforcement Learning (Elfwing et al., https://arxiv.org/abs/1702.03118) and Swish: 
+    a Self-Gated Activation Function (Ramachandran et al., https://arxiv.org/abs/1710.05941v1)
+    where the SiLU was experimented with later.
+    """
+    return x * torch.sigmoid(x)
+
+
+if torch.__version__ < "1.7":
```

Comment:
I think we can just have the other part of the test (so `if torch.__version__ >= "1.7"`) since the if clause does not change anything.